src/app/pages/MarginTradePage/components/TxFeeCalculator/index.tsx

```diff
@@ -49,7 +50,10 @@ export function TxFeeCalculator(props: Props) {
               loading={loading}
               tooltip={
                 <>
-                  {fromWei(value)} {props.symbol}
+                  {props.txConfig?.gas
+                    ? Number(props.txConfig?.gas) * Number(gas.get())
+                    : fromWei(value)}{' '}
```

Comment:
Would like to see calculations inside useMemo and outside render, also use bignumber for math related stuff - these numbers may be very big and FE may crash because of it